src/engine/main/CMakeLists.txt

```diff
@@ -173,19 +173,24 @@ if(VISIT_PARALLEL)
                     cognomen)
 
     if(WIN32)
-        set_target_properties(engine_par PROPERTIES OUTPUT_NAME enginelib_par)
+        visit_patch_target(NAME engine_par OUTPUT_NAME enginelib_par)
+    else()
+        # This keeps comm's exceptions visible when using -fvisibility=hidden
```

Comment:
Just a curiosity...why is keeping comm's exceptions visible needed/important?

Reply:
I am not sure, it was added when we moved to using CMake for the build system. Perhaps there was similar in the autoconf way of doing things way back when?